contracts/lease/src/position/mod.rs

```diff
@@ -781,3 +816,94 @@ mod test_check {
         Position::new(asset, liability, min_asset, min_sell_asset)
     }
 }
+
+#[cfg(test)]
+mod test_validate_close {
+    use currency::{lpn::Usdc, test::Dai};
+    use finance::{
+        coin::Coin,
+        duration::Duration,
+        liability::Liability,
+        percent::Percent,
+        price::{self, Price},
+    };
+
+    use crate::error::ContractError;
+
+    use super::Position;
+
+    type TestCurrency = Dai;
+    type TestLpn = Usdc;
+
+    #[test]
+    fn too_small_amount() {
+        let spec = position(100.into(), 75.into(), 15.into());
+        let result = spec.validate_close_amount(14.into(), price(1.into(), 1.into()));
```

Comment:
[all tests] Using 1:1 price all the time tests just one set of cases. I would recommend replacing it with another easy to compute value, e.g. 2:1

Reply:
TODO as a task for PR#177 - Improve tests for check_liability.